cartridges/app_custom_pd/cartridge/static/default/css/experience/components/commerceAssets/categoryComponent.css

```diff
@@ -0,0 +1,70 @@
+.category-img {
+  -o-object-fit: fill;
+     object-fit: fill;
+}
+
+.tile-container {
+  position: relative;
+}
+
+.cta-link {
+  position: absolute;
+  left: 50%;
+  -webkit-transform: translateX(-50%);
+          transform: translateX(-50%);
+  background-color: rgba(255, 255, 255, 0.5);
+  padding: 0rem 0.4rem;
+  border-radius: 0.2rem;
+  border: 1px solid #000;
```

Comment:
Use color variable instead

Reply:
That is a static file and i found a mistake in my gitignore file, these static files shouldn`t be part of the repo